salt/modules/cmdmod.py

```diff
@@ -2619,13 +2619,27 @@ def powershell(cmd,
         saltenv='base',
         use_vt=False,
         password=None,
+        depth=None,
         **kwargs):
     '''
-    Execute the passed PowerShell command and return the output as a string.
+    Execute the passed PowerShell command and return the output as a dictionary.
```

Comment:
This doesn't look accurate. This is executing a `cmd.run`, which most certainly does _not_ return a dictionary. There is a difference between a dictionary, which is a data structure, and a string of JSON, whichis not. JSON looks like a Python dictionary, but it is not a dictionary. Please change this back to what it was originally, or at least clarify and say "return the output in JSON format".